From ff011bdb13865eb22239c0114a8730c03fac04e1 Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@nxp.com>
Date: Mon, 25 Jun 2018 11:12:15 +0800
Subject: [PATCH 4121/5242] MLK-18615-2 ARM64: dts: change the camera's MCLK
 to 24MHz

commit  4a3c21987548b8de7986a8836511762abf66677d from
https://source.codeaurora.org/external/imx/linux-imx.git

change MCLK to 24MHz and change the hs_settle to 13

Signed-off-by: Robby Cai <robby.cai@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
index 7a0d314..ad3b6aa 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
@@ -706,7 +706,7 @@
 		mipi1_sensor_ep: endpoint1 {
 			remote-endpoint = <&ov5640_mipi1_ep>;
 			data-lanes = <2>;
-			csis-hs-settle = <5>;
+			csis-hs-settle = <13>;
 			csis-clk-settle = <2>;
 			csis-wclk;
 		};
@@ -762,11 +762,11 @@
 		clock-names = "csi_mclk";
 		assigned-clocks = <&clk IMX8MM_CLK_CLKO1_SRC>,
 				  <&clk IMX8MM_CLK_CLKO1_DIV>;
-		assigned-clock-parents = <&clk IMX8MM_SYS_PLL1_200M>;
-		assigned-clock-rates = <0>, <25000000>;
+		assigned-clock-parents = <&clk IMX8MM_CLK_24M>;
+		assigned-clock-rates = <0>, <24000000>;
 		csi_id = <0>;
 		pwn-gpios = <&gpio1 7 GPIO_ACTIVE_HIGH>;
-		mclk = <25000000>;
+		mclk = <24000000>;
 		mclk_source = <0>;
 		port {
 			ov5640_mipi1_ep: endpoint {
-- 
1.7.9.5

